draft-ietf-jsonpath-base.md

```diff
@@ -1104,6 +1094,62 @@ Some examples:
 | `[3,4,5]` | `$[?index(@)==2]`<br>`$[?index(@)==17]` | `[5]`<br>`[]` | Select array element |
 | `{"a":{"b":{5},c:0}}` | `$[?@.b==5 && !@.c]` | `[{"b":{5},c:0}]` | Existence  |
 
+
+## Normalized Paths
+
+A Normalized Path identifies a node by providing a query that results in exactly that node. For example,
+the JSONPath expression `$.book[?(@.price<10)]` could select two values with Normalized Paths
+`$['book'][3]` and `$['book'][5]`.
+
+An example application of Normalized Paths is a JSONPath implementation which outputs Normalized Paths
+instead of, or in addition to, the values identified by these paths.
+
+Since bracket notation is more general than dot notation, it is used to construct Normalized Paths.
+
+Single quotes are used to delimit string member names. This reduces the number of characters which
+need escaping when Normalized Paths appear as strings (which are delimited with double quotes) in JSON documents.
+
+The syntax of Normalized Paths is restricted so that there is one and only one way of representing any
+given Normalized Path. Certain characters are escaped in one and only one way; all other
+characters are unescaped.
+
+~~~~ abnf
+normalized-path           = root-selector *(S normal-index-selector)
```

Comment:
Do we actually want to allow spaces here?
That makes the output non-deterministic again.

Reply:
Nope. Good catch.

Reply:
I do agree to **not** allow them.